includes/class-sensei-learner.php

```diff
@@ -377,6 +377,8 @@ public function get_enrolled_courses_query_args( $user_id, $base_query_args = []
 			'order'       => $order,
 			'orderby'     => $orderby,
 			'tax_query'   => [], // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_tax_query -- Just empty to set array.
+			'lazy_load_term_meta' => false,
+			'cache_results' => false,
```

Comment:
Without this the terms for the post are processed and it means a query for all the terms in a post is made.